src/Service.Tests/Unittests/RuntimeConfigLoaderJsonDeserializerTests.cs

```diff
@@ -645,6 +652,7 @@ private static bool TryParseAndAssertOnDefaults(string json, out RuntimeConfig p
             Assert.IsFalse(parsedConfig.IsDevelopmentMode());
             Assert.IsTrue(parsedConfig.IsStaticWebAppsIdentityProvider);
             Assert.IsTrue(parsedConfig.IsRequestBodyStrict);
+            Assert.IsTrue(parsedConfig.IsLogLevel() is null);
```

Comment:
here you can see other examples of naming using prefix "is". They are all boolean values.

Reply:
Changing IsLogLevel to IsLogLevelNull